src/base/os.lua

```diff
@@ -610,6 +610,47 @@
 
 
 
+---
+-- Translate decorated command paths into their OS equivalents.
+---
+
+	function os.translateCommandsAndPaths(cmds, basedir, location, map)
+		local translatedBaseDir = path.getrelative(location, basedir)

```

Comment:
I don't think it matters much, whatever we prefer... it just moves this single line into 5-6 locations instead...